paddle/scripts/docker/build.sh

```diff
@@ -15,36 +15,32 @@ mkdir -p /paddle/dist/gpu-noavx
 if [ ${WITH_GPU} == "ON" ]; then
   BASE_IMAGE="nvidia/cuda:7.5-cudnn5-runtime-ubuntu14.04"
   # additional packages to install when building gpu images
-  GPU_DOCKER_PKG="python-pip"
+  GPU_DOCKER_PKG="python-pip python-dev"
```

Comment:
Thanks to this file, PR https://github.com/PaddlePaddle/Paddle/pull/1661 is sent according to this file.
Now this change may have conflict with develop branch. You can discard the change of this file, since it's already reflected in the other PR.